### ISSUE TITLE
Don't use `systemctl set-environment` for sharing settings as it leaks stuff globally to the system.

### ISSUE THREAD

**Issue by BenB196** [inverse-inc/packetfence#6698]
**Describe the bug**
Currently `systemctl set-environment` is used ([here](https://github.com/inverse-inc/packetfence/blob/0b1f7e5ac40004c8cfd62897ff8a33f363540982/lib/pf/services/manager/fingerbank_collector.pm#L38-L39) and [in this file here](https://github.com/inverse-inc/packetfence/blob/0b1f7e5ac40004c8cfd62897ff8a33f363540982/lib/pf/services/manager/fingerbank_collector.pm#L40) (Note: This file is a fingerbank file, but I wasn't able to find a repo to open this with so I'm mentioning it here as they're both Inverse products)). By using `systemctl set-environment` this causes these environment variables to be set globally to all systemd services. While this is inheritly not a bad this, it becomes a bad thing when things like `FINGERBANK_API_KEY=<snipped>` can be read by any service running on the server. These environment variables should be managed at the per service level rather than globally.

**To Reproduce**
1. Install PacketFence and Run it
2. Run: `systemctl show-environment`
3. Get a list of globally set environment variables:
```bash
COLLECTOR_ARP_LOOKUP=false
COLLECTOR_CLUSTERED=true
COLLECTOR_CLUSTER_RESYNC_INTERVAL=120s
COLLECTOR_CONTACT_PEERS=<snipped>
COLLECTOR_DB_PERSISTENCE_INTERVAL=60s
COLLECTOR_DELETE_INACTIVE_ENDPOINTS=168h
COLLECTOR_ENDPOINTS_CACHE_PATH=/usr/local/fingerbank/db/collector_endpoints_cache.db
COLLECTOR_ENDPOINTS_DB_PATH=/usr/local/fingerbank/db/collector_endpoints.db
COLLECTOR_HTTP_CERT=/usr/local/pf/conf/ssl/server.crt
COLLECTOR_HTTP_KEY=/usr/local/pf/conf/ssl/server.key
COLLECTOR_IP_MAPS_DB_PATH=/usr/local/fingerbank/db/collector_ip_maps.db
COLLECTOR_NETWORK_BEHAVIOR_ANALYSIS=false
COLLECTOR_QUERY_CACHE_TIME=1440m
FINGERBANK_API_KEY=<snipped>
GODEBUG=gctrace=0
GOGC=100
HTTP_PROXY=
LANG=en_US.UTF-8
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin
PORT=4723
```
4. If you check any no-PacketFence related systemd service they will also have these variables.
5. Using SSH as an example:
```bash
# ps aux | grep ssh
root     24679  0.0  0.0 117124  4176 ?        Ss   Jun30   0:00 /usr/sbin/sshd -D -u0
```
6. Get environment variables of the pid:
```bash
cat /proc/24679/environ | tr '\0' '\n'
```
7. See that it has the globally set variables:
```bash
# cat /proc/24679/environ | tr '\0' '\n'
COLLECTOR_ARP_LOOKUP=false
COLLECTOR_CLUSTERED=true
COLLECTOR_CLUSTER_RESYNC_INTERVAL=120s
COLLECTOR_CONTACT_PEERS=<snipped>
COLLECTOR_DB_PERSISTENCE_INTERVAL=60s
COLLECTOR_DELETE_INACTIVE_ENDPOINTS=168h
COLLECTOR_ENDPOINTS_CACHE_PATH=/usr/local/fingerbank/db/collector_endpoints_cache.db
COLLECTOR_ENDPOINTS_DB_PATH=/usr/local/fingerbank/db/collector_endpoints.db
COLLECTOR_HTTP_CERT=/usr/local/pf/conf/ssl/server.crt
COLLECTOR_HTTP_KEY=/usr/local/pf/conf/ssl/server.key
COLLECTOR_IP_MAPS_DB_PATH=/usr/local/fingerbank/db/collector_ip_maps.db
COLLECTOR_NETWORK_BEHAVIOR_ANALYSIS=false
COLLECTOR_QUERY_CACHE_TIME=1440m
FINGERBANK_API_KEY=<snipped>
GODEBUG=gctrace=0
GOGC=100
HTTP_PROXY=
LANG=en_US.UTF-8
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin
PORT=4723
```

**Expected behavior**
Environment variables specific to PacketFence and its subprocesses should not leak into the global system hosting it.

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

**Additional context**
PacketFence 10.3 3 Node Cluster

**Comment by julsemaan** [comment]
It has been pushed to `packetfence-1.3` in the collector's repo

The next collector release will contain it, will close this issue once its released

**Comment by julsemaan** [comment]
Released in 1.3.18 available for PacketFence 11.0 and above
